ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -28,8 +34,10 @@ export interface AccountValue {
   proofAuthorizingKey: string | null
 }
 
-export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
-  serialize(value: AccountValue): Buffer {
+export type AccountValue = EncryptedAccountValue | DecryptedAccountValue
+
+export class AccountValueEncoding implements IDatabaseEncoding<DecryptedAccountValue> {
```

Comment:
This is a temporary change to update the generic and will be changed back to `AccountValue` in a subsequent PR. We are using `DecryptedAccountValue` to keep code changes small per PR, and the change will come once the wallet and walletdb are updated to read / store encrypted accounts.